### PR TITLE
add init template for community

### PR DESCRIPTION
**Pull request by edisonxiang**
add init template for community

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
+# Project Name
```
Comment (TommyLike) [inline README.md]: This one is README_TEMPLATE file. right?

File: INSTALL.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
+# How to install
```
Comment (TommyLike) [inline INSTALL.md]: This file is INSTALL_TEMPLATE file?

Comment (edisonxiang) [reply INSTALL.md]: It is not good to rename file name.

File: README.md (hunk 1 above)
Comment (edisonxiang) [reply README.md]: dito

**Pull request merged**